controllers/flinkcluster_reconciler.go

```diff
@@ -103,6 +104,11 @@ func (reconciler *ClusterReconciler) reconcile() (ctrl.Result, error) {
 		return ctrl.Result{}, err
 	}
 
+	err = reconciler.reconcileHPA()
```

Comment:
Can we put HPA at the end? because if it fails, other reconciliation can be done unaffected.

Reply:
Good idea, changing this